### PR TITLE
Redirect the user to the wizard page if no clusters

### PR DESCRIPTION
**Pull request by David-DB88**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

Redirect the user to the wizard page if no clusters

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -55,6 +57,12 @@ const Dashboard: React.FC = () => {
     return initialColumns;
   }, []);
 
+  useEffect(() => {
+    if (!clusters && appInfo.hasDynamicConfig) {
+      navigate(getNonExactPath(clusterNewConfigPath));
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Dashboard/Dashboard.tsx:62]: why non exact?

Comment (David-DB88) [reply kafka-ui-react-app/src/components/Dashboard/Dashboard.tsx:62]: understood

File: kafka-ui-react-app/src/components/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -55,6 +57,12 @@ const Dashboard: React.FC = () => {
     return initialColumns;
   }, []);
 
+  useEffect(() => {
+    if (!clusters && appInfo.hasDynamicConfig) {
```
[the remark this answers is not in the document]
Comment (David-DB88) [reply kafka-ui-react-app/src/components/Dashboard/Dashboard.tsx:61]: done

File: kafka-ui-react-app/src/components/Dashboard/Dashboard.tsx (hunk 1 above)
Comment (David-DB88) [reply kafka-ui-react-app/src/components/Dashboard/Dashboard.tsx]: removed it

**Review by workshur — APPROVED** [review]

File: kafka-ui-react-app/src/components/Dashboard/__test__/Dashboard.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import Dashboard from 'components/Dashboard/Dashboard';
+import { Cluster, ServerStatus } from 'generated-sources';
+import { render } from 'lib/testHelpers';
+
+interface DataType {
+  data: Cluster[];
+  isSuccess: boolean;
+}
+jest.mock('lib/hooks/api/clusters');
+const mockedUsedNavigate = jest.fn();
```
[the remark this answers is not in the document]
Comment (David-DB88) [reply kafka-ui-react-app/src/components/Dashboard/__test__/Dashboard.spec.tsx:12]: done

**Review by Mgrdich — APPROVED** [review]

**Review by Haarolean — APPROVED** [review]

**Pull request merged**